src/app/legacy/psammead-section-label/src/index.jsx

```diff
@@ -48,6 +48,7 @@ const Heading = styled.h2`
   /* reset default margins */
   margin: 0;
   padding: 0;
+  font-weight: bold;
```

Comment:
Looks good! Just thinking, do we need to use the `getSansBold()` utility here to determine the font weight? I've seen it used elsewhere, so it may need use here instead of `font-weight:bold`

Reply:
I think it could be this line that needs updated to use the `getSansBold`:
https://github.com/bbc/simorgh/blob/238a71e92baa22f9d73196ce742587fbeb88ef7f/src/app/legacy/psammead-section-label/src/titles.jsx#L78

Its quite a nested component, so I think that should target the element wrapping the text.